tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -24,6 +24,32 @@ class TestPVCCreationPerformance(E2ETest):
     """
     pvc_size = '1Gi'
 
+    def samples_creation(self, samples_num, teardown_factory, pvc_size):
+        """
+        samples_creation creates number (samples_num) of PVCs, measures creation time for each PVC and returns list
+        with all the creation times
+        :param pvc_size: Size of the created PVC
+        :param teardown_factory: A fixture used when we want a new resource that was created during the tests
+        :param samples_num: Number of the sampled created PVCs
+        :return: list of the creation times of all the created PVCs
+        """
+        time_measures = []
+        for i in range(samples_num):
+            log.info(f'Start creation of PVC number {i + 1}.')
+
+            pvc_obj = helpers.create_pvc(
+                sc_name=self.sc_obj.name, size=pvc_size)
+            helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+            pvc_obj.reload()
+            teardown_factory(pvc_obj)
+            create_time = helpers.measure_pvc_creation_time(
```

Comment:
Isn't the intention here to measure the PVC creation time? if so, why is the measurement being taken post `teardown_factory` call?

Reply:
Also, can you please check if `tests.helpers.measure_pvc_creation_time` suites you? If so, maybe this function is not needed

Reply:
@ebenahar  -
> Isn't the intention here to measure the PVC creation time? if so, why is the measurement being taken post `teardown_factory` call?
I was not sure whether the call for the teardown_factory should be counted as part of pvc creation time. 
In the test as it was before my modification, the code was exactly like this ( teardown_factory called before the measure was taken).

Reply:
@ebenahar 


> Also, can you please check if `tests.helpers.measure_pvc_creation_time` suites you? If so, maybe this function is not needed

tests.helpers.measure_pvc_creation_time definitely will not suit me, since I need to create and measure times for scalable number of PVCs. 
In case you referred to measure_pvc_creation_time function: 
It measures times for existing PVCs which is not the case here, since the function I've added also creates the PVCs.

Reply:
1) pvc creation time is measured from the logs, so I don't think it matters whether it is done after the teardown
2) isn't `measure_pvc_creation_time` actually the function used here?

Reply:
@keesturam  I use measure_pvc_creation_time function inside the new create_mutiple_pvcs_statistics function. 
I think that Elad was asking why to write this function in the first place, when we have the measue_pvc_creation_time function. 
My reply was - we need a new function not only for measuring existing PVCs but to create number ( scaleable) of PVCs and then to return statistics on their creation times.